src/utils/sendMessages.js

```diff
@@ -0,0 +1,31 @@
+const { promisify } = require('util');
+
+const sleep = promisify(setTimeout);
+
+/**
+ * Slack has a 50 block limit, per message. We need to chunk blocks so we can send them
+ * in separate messages if there are more than 50 blocks in a message.
+ * More here: https://api.slack.com/reference/block-kit/blocks
+ *
+ * This function will send out one message per second, until all messages have
+ * been sent.
+ *
+ * @param {Function} sender function that sends a message via slack api
+ * @param {Array} blocks Slack "blocks" to send via message
+ */
+const sendDelayedMessages = async function sendMessages(sender, blocks) {
+  await Promise.all(
```

Comment:
Do we want to guarantee the sending order here? It might a good idea to build up a chain of promises if we want to ensure messages are sent in the correct order.

Reply:
Yes, that is something I ran into during testing. I'd be open to a suggestion of how to approach this.

Reply:
@therynamo A traditional `for` loop would work I think, or a `for ... of`:
```ts
for (const [blockChunk, i] of blocks.entries()) {
  // The slack rate limit for messages is 1 per second, unless it is defined in
  // "special rate limits".
  if (i !== 0) await sleep(1500);
  try {
    sender(blockChunk);
  } catch(error) {
    console.log(error);
  }
}
```

This might be a bit slower, but correctness would be guaranteed.:tm:

Reply:
Let's do it! I'll put up a PR to test it out.